openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/ProfilerTab.test.tsx

```diff
@@ -0,0 +1,142 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { act, cleanup, render, screen } from '@testing-library/react';
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { getListTestCase } from '../../../axiosAPIs/testAPI';
+import { Column } from '../../../generated/entity/data/table';
+import {
+  COLUMN_PROFILER_RESULT,
+  MOCK_TABLE,
+  TEST_CASE,
+} from '../../../mocks/TableData.mock';
+import { ProfilerTabProps } from '../profilerDashboard.interface';
+import ProfilerTab from './ProfilerTab';
+
+const profilerTabProps: ProfilerTabProps = {
+  profilerData: COLUMN_PROFILER_RESULT,
+  activeColumnDetails: MOCK_TABLE.columns[0],
+  tableProfile: MOCK_TABLE.profile,
+};
+
+jest.mock('../../../axiosAPIs/testAPI', () => {
+  return {
+    getListTestCase: jest
+      .fn()
+      .mockImplementation(() => Promise.resolve(TEST_CASE)),
+  };
+});
+
+jest.mock('./ProfilerSummaryCard', () => {
+  return jest.fn().mockImplementation(() => <div>ProfilerSummaryCard</div>);
+});
+
+jest.mock('./ProfilerDetailsCard', () => {
+  return jest.fn().mockImplementation(() => <div>ProfilerDetailsCard</div>);
+});
+
+jest.mock('react-i18next', () => ({
+  // this mock makes sure any components using the translate hook can use it without a warning being shown
+  useTranslation: () => {
+    return {
+      t: (str: string) => str,
+    };
+  },
+}));
+
+describe('Test ProfilerTab component', () => {
+  beforeEach(() => cleanup());
```

Comment:
We don't need this one as it's mentioned globally [here](https://github.com/open-metadata/OpenMetadata/blob/c4b7432c073f080fbd668c583f9a6a170ecb96b2/openmetadata-ui/src/main/resources/ui/jest.config.js#L45)

Reply:
Oh ok, Thanks for highlighting 🙏🏻